README.md

```diff
@@ -20,7 +20,7 @@ This will add the helper module to your `package.json` file as a development dep
 ...
 ```
 
-The test-helper requires the node-red runtime to run its tests, but Node-RED is **not** installed as a dependency.  The reason for this is that test-helper is (or will be) used in Node-RED core tests, and Node-RED itself has a large number of dependencies that you may not want to download if you already have it installed.
+The test-helper requires the node-red runtime, but Node-RED is **not** installed as a dependency.  The reason for this is that test-helper is (or will be) used in Node-RED core tests, and Node-RED itself has a large number of dependencies that you may not want to download if you already have it installed.
```

Comment:
I'm not sure the intended average user of this module needs to the details about know why not.  That's documentation which could be better kept elsewhere, IMO.

Reply:
Also the proper names here are inconsistent; we should probably mention `peerDependencies` or "peer dependency" right off the bat.

Reply:
Ok.  Will remove and add any useful info here to the 'creating a node' documentation at some point.